code/modules/zombie/items.dm

```diff
@@ -99,10 +94,8 @@
 	user.Weaken(30)
 	var/success = offer_control(user)
 	if(success)
-		user.visible_message("<span class='suicide'>[user] appears to have \
-			found new spirit.</span>")
+		user.visible_message("<span class='suicide'>[user] appears to have found new spirit.</span>")
```

Comment:
Except that's not what it's for, it's because the player has been replaced by another player when they were offered control.

Found a new spirit is obviously the intended value